src/main/java/demo/TheFoo.java

```diff
@@ -5,7 +5,7 @@
 import lombok.RequiredArgsConstructor;
 import lombok.ToString;
 
-import javax.annotation.Nonnull;
+import jakarta.annotation.Nonnull;
```

Comment:
The `javax` namespace [has mostly migrated now](https://jakarta.ee/blogs/javax-jakartaee-namespace-ecosystem-progress/) to `jakarta`.